fe/fe-core/src/main/java/com/starrocks/catalog/TableProperty.java

```diff
@@ -142,6 +146,12 @@ public TableProperty buildReplicationNum() {
         return this;
     }
 
+    public TableProperty buildPartitionTTL() {
+        partitionTTLNumber = Integer.parseInt(properties.getOrDefault(PropertyAnalyzer.PROPERTIES_PARTITION_TTL_NUMBER,
```

Comment:
how about we just call the property `partition_ttl`, normally we don't add `_number` to it

Reply:
partition_ttl_number means how many partitions to keep, this number cannot be omitted.